src/controllers/student.controls.js

```diff
@@ -1,4 +1,5 @@
-const Model = require('../database/models/students.model');
+const studentModel = require('../database/models/students.model');
+const courseModel = require('../database/models/courses.model');
```

Comment:
I would keep using PascalCase for these, which is typically the convention for naming class like objects
`StudentModel` and `CourseModel`